spec.html

```diff
@@ -27605,7 +27605,7 @@ <h1>Properties of the Number Constructor</h1>
 
       <emu-clause id="sec-number.epsilon">
         <h1>Number.EPSILON</h1>
-        <p>The value of `Number.EPSILON` is the difference between 1 and the smallest value greater than 1 that is representable as a Number value, which is approximately 2.2204460492503130808472633361816 x 10<sup> - 16</sup>.</p>
+        <p>The value of `Number.EPSILON` is the difference between 1 and the smallest value greater than 1 that is representable as a Number value, which is approximately 2.2204460492503130808472633361816 &times; 10<sup>-16</sup>.</p>
```

Comment:
The ` - 16` -> `-16` change affects the rendering, but I consider it to be an improvement. It also makes this match what we do in Number.MIN_VALUE, which we describe as being approximately `5 &times; 10<sup>-324</sup>`.

Reply:
I very much agree this is an improvement
https://tc39.es/ecma262/#sec-number.epsilon vs https://ci.tc39.es/preview/tc39/ecma262/sha/285329afcf5011293bbf5b85f72beff1e8b54d21/#sec-number.epsilon